memory-bank/productContext.md

```
@@ -0,0 +1,147 @@
+
+## Domain: Planning Poker Estimation
+
+### What is Planning Poker?
+
+Planning poker (also called Scrum poker) is a consensus-based, gamified technique for estimating effort or relative size of development goals in software development. Team members make estimates by playing numbered cards face-down, then revealing them simultaneously to avoid anchoring bias.
+
+### User Flows
+
+#### 1. Moderator Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)